src/assets/scss/_global.scss

```diff
@@ -0,0 +1,4 @@
+@import "0-vendor/__vendor";
```

Comment:
Thought: `andculturecode-javascript-react-components.scss` is more of a true "global" import file. Definitely up for suggestions, but recommend renaming and moving this file to ...

`assets/scss/6-components/_component-base.scss` with potential contents like so...

```scss
/*------------------------------------*\
  Component base SCSS import file
  -------------------------------
  Imported directly into individual
  component SCSS files. Anything added
  is imported into every component.
\*------------------------------------*/
@import "0-vendor/__vendor";
@import "1-settings/__settings";
@import "2-tools/__tools";
@import "3-generic/__generic";
```

Reply:
I can get on board with this, but do we really want to import `3-generic/__generic` ?  Is this really something we want to do?  Consider a project that uses a single react component from this repo.  By now using it, we've now overriden all the consumer's default html and body element styling.  I feel that's something that is something that should still be set on a per project basis.

Reply:
Definitely hear what you are saying. Overriding defaults is one of the "challenges" regardless of our approach to placement of SCSS files. The current implementation directs consumers to load the top-level/global SCSS file (andculturecode-javascript-react-components) to get _everything_ and from there consumers override parts.

Regardless of which direction we take, the future for this repo's provided vendor, settings, tools, generic, etc... will likely need adjusted and further reduced for the masses. There are aspects of __generic that we setup almost every time that _would_ benefit us. While yes, in a literal sense it only serve storybook for _this_ repo, these styles are not just intended for storybook. Our boilerplate and other consuming projects use this for both the application itself and storybook.